sdk/purview/Azure.Analytics.Purview.Scanning/src/Azure.Analytics.Purview.Scanning.csproj

```diff
@@ -8,6 +8,7 @@
     <NoWarn>$(NoWarn);AZC0001;CS0169</NoWarn>
     <DefineConstants>$(DefineConstants);EXPERIMENTAL</DefineConstants>
     <IncludeOperationsSharedSource>true</IncludeOperationsSharedSource>
+    <DocumentationFile>docs/Azure.Analytics.Purview.Scanning.xml</DocumentationFile>
```

Comment:
if we don't care about the generated xml file, then we only need to add `<GenerateDocumentationFile>true</GenerateDocumentationFile>`.